Yoast/wpseo-woocommerce
# Make the versioning setup the same as news seo

## Summary

This PR can be summarized in the following changelog entry:

* N/A - Tooling - The version update command didn't work.

## Relevant technical choices:

* Decided to make an extra constant that feeds the value of class constant `Yoast_WooCommerce_SEO::version` - We should remove the last one when the time is there.
* I also removed a faulty config that relies on an old location. I guess the class has been moved to another file in the meanwhile.

## Test instructions

This PR can be tested by following these steps:

* Set a new version and run the update-version command
* The version should be updated now.

Fixes #491
